OPass/Tabs/CheckinTab/CheckinCardView.swift

```diff
@@ -53,7 +53,7 @@ class CheckinCardView: UIView {
         return self.scenario!
     }
 
-    func buttonUpdate(_ intermediate: (() -> ())?, _ completeion: (() -> ())?, _ cleanup: (() -> ())?) {
+    func buttonUpdate(_ intermediate: (() -> Void)?, _ completeion: (() -> Void)?, _ cleanup: (() -> Void)?) {
```

Comment:
I don’t really sure that is necessary to change `() -> ()` to `() -> Void`, it is equally and short in before

Reply:
they are actually the same, in headers they declare  `typealias Void = ()`
but I think `Void` is more readable than `()`

Reply:
Okay, can lint force tricks `() -> ()` as a warning for using `() -> Void` instead?

Reply:
it could but I believe that we should only have one style in project, so if `Void` is unacceptable then I will turn move back to `() -> ()`

Reply:
just set one style for whole project, if you can set only accept `() -> Void`, just do it, otherwise let it keep simple

Reply:
yes currently I only set `() -> Void` is valid

Reply:
it looks like `void_return` is right?

Reply:
confirmed via IM, perform to merge this PR